table/server/common/src/main/java/alluxio/table/common/udb/UdbConfiguration.java

```diff
@@ -83,6 +84,15 @@ public UdbConfiguration(Map<String, String> values) {
       // include the trailing '/'
       schemeAuthority += "/";
     }
-    return mMountOptions.getOrDefault(schemeAuthority, Collections.emptyMap());
+    Map<String, String> map =
+        mMountOptions.getOrDefault(schemeAuthority, Collections.emptyMap());
+    if (map.equals(Collections.emptyMap())) {
+      for (Entry<String, Map<String, String>> entry : mMountOptions.entrySet()) {
+        if (schemeAuthority.matches(entry.getKey())) {
+          return entry.getValue();
+        }
+      }
+    }
```

Comment:
Could treating authority strings as regex expressions cause ambiguity? e.g. "example.com" can be a string literal and a regex expression matching "example/com" at the same time.

Reply:
Good catch, thank you for pointing this out.